policyengine_us/variables/gov/usda/snap/snap_unit.py

```diff
@@ -0,0 +1,15 @@
+from policyengine_us.model_api import *
+
+
+class snap_unit(Variable):
```

Comment:
`snap_unit` implies an entity. Which we might need?
```suggestion
class snap_unit_size(Variable):
```